content/birds/miriam.md

```diff
@@ -12,10 +12,11 @@ image:
     and white-blue-pink lightning-bolt earrings
   position: top
 social:
-  twitter: TerribleMia
```

Comment:
This replaces `@TerribleMia` with `@oddbird` in the `<meta name="twitter:creator">` tag on all your posts, but that's probably fine?

Reply:
Yeah, I suppose that's ok.